mne/io/eeglab/eeglab.py

```diff
@@ -84,6 +82,8 @@ def _get_info(eeg, montage, eog=()):
             montage = Montage(np.array(pos), ch_names, kind, selection)
     elif isinstance(montage, string_types):
         path = op.dirname(montage)
+    else:  # if eeg.chanlocs is empty, we still need default chan names
+        ch_names = ["EEG " + str(ii) for ii in range(eeg.nbchan)]
```

Comment:
ch_names = ["EEG %03d" % ii for ii in range(eeg.nbchan)]

so it renders EEG 001, EEG 002, etc...